nexus/src/app/background/inventory_collection.rs

```diff
@@ -180,7 +181,11 @@ impl<'a> nexus_inventory::SledAgentEnumerator for DbSledAgentEnumerator<'a> {
             while let Some(p) = paginator.next() {
                 let records_batch = self
                     .datastore
-                    .sled_list(&self.opctx, &p.current_pagparams())
+                    .sled_list(
+                        &self.opctx,
+                        &p.current_pagparams(),
+                        SledFilter::All,
```

Comment:
No objection on this PR, but this might not be right in the limit. (Some discussion in chat after #5480 noted inventory collection timeouts trying to talk to expunged sleds.)

Reply:
Yeah, I think based on the level of discussion that should be left to a follow up.